README.md

```diff
@@ -205,4 +205,6 @@ Blender: Original by Benjamin Amos
 
 Fried Chicken: https://pixabay.com/en/chicken-fried-leg-piece-poultry-37807/
 
-Chicken: http://www.clipartpanda.com/clipart_images/shared-by-leonnardo00-9075011
\ No newline at end of file
+Chicken: http://www.clipartpanda.com/clipart_images/shared-by-leonnardo00-9075011
+
+Toast: http://piq.codeus.net/picture/55650/toast (edited)(resized)
```

Comment:
Since it looks like the art was made in 2009 and the user is inactive and I couldn't find it's licensing policies, I just made a texture for toast myself